docs/API.md

```diff
@@ -446,13 +446,16 @@ mysql2
 | http.status_code | The HTTP status code of the response.                     |
 | http.headers.*   | A recorded HTTP header.                                   |
 
-##### Configuration Options
+#### sails
```

Comment:
This removes the restify configuration options by mistake?  Also looks like their branding prefers capital S.

Reply:
My mistake, I will add that back in. Thanks

Do we follow branding in any of the other section titles? It appears to be just the name of the package for the other ones, but if you prefer proper names instead, we should change all of them.

Reply:
fair point.  I'll update the others in my api doc changes.